client/src/components/UserContacatCard/index.js

```diff
@@ -0,0 +1,73 @@
+import {
+  Card,
+  CardContent,
+  Typography,
+  Avatar,
+  colors,
+
+} from '@mui/material';
+import MdPhone from '@mui/icons-material/Phone';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import EmailIcon from '@mui/icons-material/Email';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  card: {
+    backgroundColor: 'secondary',
+    maxWidth: 345,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '10px',
+  },
+  icon: {
+    marginRight: '0.5rem',
+  },
+});
+const contentStyles = { display: 'flex', marginBottom: '15px', alignItems: 'center' };
+function UserContactCard({
+  image, name, location, phone, email,
```

Comment:
I think it's better if we pass data object here and destructure it here and use it so we could use this component like this 
```jsx
<Card data={data} />
```
instead of 
```jsx
<Card name={data.name} location={data.location} phone={data.phone} email={data.email} image={data.image} />
```